src/components/PriceRange.vue

```diff
@@ -69,7 +68,7 @@ export default {
     },
     from: {
       get() {
-        for (let refinement in this.searchStore.activeRefinements) {
+        for (const refinement in this.searchStore.activeRefinements) {
```

Comment:
Also we should not use for const in on arrays but prefer for of or array methods